### PR TITLE
Added Adversary Sheet

### PR DESCRIPTION
**Pull request by stuttste**
I needed a more fleshed out adversary sheet, so went ahead and implemented it. May tackle delve/landmark if I feel the need.

### CODE REVIEW

File: src/actors/adversary/sheet.html [hunk 1]
```diff
@@ -1 +1,100 @@
...
+    {{/if}}
+</div>
+{{/inline}}
+
+<div>
+<form class="heart sheet actor adversary {{cssClass}}" autocomplete="off">
+        <img class="profile" src="{{data.img}}" title="{{actor.name}}" data-edit="img"/>
+
+
+        <div class="section grid-3 resistances place-top">
```
Comment (hitcherland) [inline src/actors/adversary/sheet.html:33]: We can delete these two divs and set the css class to grid-2

File: release-notes/v0.4.6/lang/en.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+    "versions": {
+        "0_4_5": {
+            "version": "v0.4.6",
+            "changes":[
+                "Created more fleshed-out adversary sheet complete with drag-n-drop resources and equipment and variable max resistance"
```
Comment (hitcherland) [inline release-notes/v0.4.6/lang/en.json:6]: Credit yourself!

File: src/actors/adversary/template.json [hunk 3]
```diff
@@ -6,10 +6,12 @@
             "motivation": "",
             "difficulty": "",
             "resistance": "",
+            "resistanceMax": "",
```
Comment (hitcherland) [inline src/actors/adversary/template.json:9]: We should set this to a default value (say 5), as it's confusing to see zero boxes when you first create an adversary.

**Comment by hitcherland** [comment]
Oh I forgot to mention, we need to add some extra translations, the `en.json` file in the adverary directory should look something like

```
{
    "adversary": {
        "label-single": "Adversary",
        "label-multiple": "Adversaries",
        "name": "Name",
        "resistanceMax": "Maximum Resistance",
        "difficulty": "difficulty",
        "descriptors": "Descriptors",
        "motivation": "Motivation",
        "special": "Special",
        "notes": "Notes",
        "inactive-items": "Inactive Items",
        "inactive-items-description": "Here you'll find items that were directly added to the adversary sheet, but were marked inactive (which hides them from the normal layout)."
    }
}
```

**Comment by stuttste** [comment]
No problem! Glad I could help. I'll take a look and try to add in the suggested changes (a bit new to using some of the more in-depth features of github like pull requests) and get them back to ya. 

I'm not super familiar with javascript/web dev, so tried not to meddle with things I wasn't sure of. More of a C# man. lol

File: release-notes/v0.4.6/lang/en.json (hunk 2 above)
Comment (stuttste) [reply release-notes/v0.4.6/lang/en.json:6]: What's the correct method of doing so? I'm not familiar with the schema of these release notes objects.

Comment (hitcherland) [reply release-notes/v0.4.6/lang/en.json:6]: Just in text I think! "Thanks to stuttste, created more ..."

**Comment by stuttste** [comment]
Ok, I *think* I got everything. Threw in Spanish translation as well because why not.

Double-check the en.json/es.json files. I added a resistance and protection entry so it would find those nicely, but not sure if it will cause any conflicts with other things in the project.

**Comment by hitcherland** [comment]
Should be available now as v0.4.6!

**Pull request merged**